examples/calculate_classify_ndvi.py

```diff
@@ -0,0 +1,149 @@
+#    The examples below will show you how to use the ``normalized_diff()`` function
+#    to calculate the normalized difference vegetation index (NDVI), a commonly
+#    used remotely sensed index for quantifying vegetation health.
+#
+# The example below walks you through a typical workflow for calculating the normalized
+# difference vegetation index (NDVI) using Landsat 8 data with EarthPy. NDVI provides
+# a measure of healthy vegetation and ranges in value from -1 to 1. Values closer to
+# 1 represent healthy, green vegetation. NDVI can be calculated from Landsat 8 data
+# using band 4 (red) and band 5 (near-infrared).
+#
+# First, you will create a stack of bands using Landsat 8 data and then calculate
+# NDVI using the ``normalized_diff()`` function. Then, you will plot the NDVI results
+# using a colorbar legend with continuous values. Last, you will classify the NDVI
+# results using threshold values and plot the classified data with a categorical legend.
+
+
+###############################################################################
+# Import Packages
+# ---------------
+#
+# To begin, import the needed packages. You will use a combination of several EarthPy
+# modules including spatial and plot.
+
+import os
+from glob import glob
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import ListedColormap
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
+
+
+###############################################################################
+# Import Example Data
+# -------------------
+#
+# To get started, make sure your directory is set. Create a stack from all of the
+# Landsat .tif files (one per band).
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Stack the Landsat 8 bands
+# This creates a numpy array with each "layer" representing a single band
+landsat_path = glob(
+    "data/cold-springs-fire/landsat_collect/LC080340322016072301T1-SC20180214145802/crop/*band*.tif"
+)
+landsat_path.sort()
+arr_st, meta = es.stack(landsat_path)
+
+
+###############################################################################
+# Calculate Normalized Difference Vegetation Index (NDVI)
+# -------------------------------------------------------
+
+# You can calculate NDVI for your dataset using the
+# ``normalized_diff`` function from the ``earthpy.spatial`` module.
+# Math will be calculated (b1-b2) / (b1 + b2).
+
+# Landsat 8 red band is band 4 at [3]
+# Landsat 8 near-infrared band is band 5 at [4]
+ndvi = es.normalized_diff(arr_st[4], arr_st[3])
+
+
+###############################################################################
+# Plot NDVI With Colorbar Legend of Continuous Values
+# ----------------------------------------------------
+
+# You can plot NDVI with a colorbar legend of continuous values using the
+# ``colorbar`` function from the ``earthpy.plot`` module.
+
```

Comment:
@jlpalomino  guess what? i've updated `plot_bands` to provide colorbars. Can you please plot using plot_bands instead?? it will be a good test to see if it works for you as well. look at the docs as i've updated the parameters and such (vmin and max are included in that update).

Reply:
Implemented and it works well!